Pod/Classes/URBNAlertView.m

```diff
@@ -353,12 +353,14 @@ - (URBNAlertActionButton *)createAlertViewButtonWithAction:(URBNAlertAction *)ac
     btn.layer.borderColor = self.alertStyler.buttonBorderColor.CGColor;
     btn.layer.borderWidth = self.alertStyler.buttonBorderWidth.floatValue;
     btn.contentEdgeInsets = self.alertStyler.buttonContentInsets;
+    
     btn.tag = index;
     btn.actionType = action.actionType;
     btn.alertStyler = self.alertStyler;
     
     [btn setTitle:action.title forState:UIControlStateNormal];
     [btn setTitleColor:titleColor forState:UIControlStateNormal];
+    [btn setTitleColor:self.alertStyler.buttonHighlightTitleColor forState:UIControlStateHighlighted];
```

Comment:
will you ever need to have a different highlight color for cancel/destructive buttons? We currently have those styles in but if thats something you wont need then theres no need to get them in now